drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/ActionSetFieldPopup.java

```diff
@@ -385,13 +386,25 @@ private void doBindingLabel() {
     void doFieldLabel() {
         if ( !nil( this.editingCol.getFactField() ) ) {
             setFieldLabelToFieldName( this.editingCol.getFactField() );
-        } else if ( !nil( this.editingCol.getBoundName() ) ) {
+            this.editField.setEnabled( true );
+        } else if ( !nil( this.editingCol.getBoundName() ) && !isChooseConstant(editingCol.getBoundName())) {
             setFieldLabelPleaseSelectAField();
+            this.editField.setEnabled( true );
         } else {
             setFieldLabelPleaseChooseAFactPatternFirst();
+            this.editField.setEnabled( false );
         }
     }
 
+    boolean isChooseConstant(String text) {
+        if ( text == null || text.isEmpty() ) {
+            return false;
+        }
+        
+        return GuidedDecisionTableConstants.INSTANCE.Choose().compareTo(text) == 0 ||
```

Comment:
BTW it is not "nice" having to perform this check, but without more extensive refactoring it is probably the easiest option.